lib/full-site-editing/templates-utils.php

```diff
@@ -5,6 +5,40 @@
  * @package gutenberg
  */
 
+/**
+ * Parses the `wp_theme` terms of a template or template part into:
+ * [
+ *   'is_file_based', // Whether the template is based upon a theme-provided file.
+ *   'is_original', // Whether the template is the original theme-provided file.
+ *   'themes', // An array of themes that this template belongs to.
```

Comment:
How can a template belong to multiple themes?

Reply:
It can't right now. But in the future we might want to share templates between multiple themes. Which would be possible by assigning it to multiple themes.

Something like that I guess.

Reply:
We will also eventually decide what to do with child themes that inherit templates from their parent.
One (naive, probably) way to handle it is to have both child and parent themes slugs in the array. 🤔